src/NLog/Targets/MailTarget.cs

```diff
@@ -650,7 +650,7 @@ private MailMessage CreateMailMessage(LogEventInfo lastEvent, string body)
             if (Priority != null)
             {
                 var renderedPriority = Priority.Render(lastEvent);
-                if (ConversionHelpers.TryParseEnum(renderedPriority, out MailPriority mailPriority))
+                if (!string.IsNullOrEmpty(renderedPriority) && ConversionHelpers.TryParseEnum(renderedPriority, out MailPriority mailPriority))
```

Comment:
Should be `if (ConversionHelpers.TryParseEnum(renderedPriority, out MailPriority mailPriority)) { ... } else if (!string.IsNullOrEmpty(renderedPriority) { Log-warning }`

Reply:
The parse is wrong if the value is empty?

Reply:
Right now it logs a warning every time `string.IsNullOrEmpty(renderedPriority)==true`

Reply:
Oops that's wrong of course. Thanks!